### PR TITLE
Update aircrack-ng.rb

### PR DESCRIPTION
**Pull request by souvik1997**
Fixes #29450

### CODE REVIEW

**Comment by DomT4** [comment]
The patch here is going to need submitting upstream, and a link to that submission included in the formula.

File: Library/Formula/aircrack-ng.rb [hunk 1]
```diff
@@ -16,11 +14,6 @@ class AircrackNg < Formula
   patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/aircrack-ng.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Comment by souvik1997** [comment]
Here's the link to the upstream patch submission: http://trac.aircrack-ng.org/ticket/1609

**Comment by mikemcquaid** [comment]
> Here's the link to the upstream patch submission: http://trac.aircrack-ng.org/ticket/1609

@souvik1997 Please link it in the formula, thanks.

**Comment by mikemcquaid** [comment]
I'd also use a `patch do` with http://trac.aircrack-ng.org/attachment/ticket/1609/aircrack-1.2rc2.patch if you can.

**Comment by souvik1997** [comment]
I added a comment to explain the purpose of the patch. I don't understand what you mean by `patch do` though.

**Comment by dunn** [comment]
Here's an example of the `patch do` form: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/avidemux.rb#L11-L14

**Comment by souvik1997** [comment]
So I tried using the `patch do` format but I can't seem to get it to work. It downloads the patch but fails to apply it. This is the output of the patch command:
```
patching file scripts/airodump-ng-oui-update
Hunk #1 succeeded at 20 with fuzz 1 (offset 13 lines).
patching file src/osdep/radiotap/platform.h
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- aircrack-ng-1.2-rc2/src/osdep/radiotap/platform.h
|+++ aircrack-ng-1.2-rc2/src/osdep/radiotap/platform.h
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
Error: Failure while executing: /usr/bin/patch -g 0 -f -p0 -i /private/tmp/aircrack-ng--patch20150821-41774-7b9x6/aircrack-ng-1.2-rc2.patch
```
I tried using p0 and p1 and the formula fails with both. However manually applying the patch using the terminal works.

**Comment by dunn** [comment]
I guess make it inline again, then. Sorry for the back-and-forth.

**Comment by bfontaine** [comment]
Ping?

**Comment by bfontaine** [comment]
Closing due to no response; feel free to reopen a PR that passes the tests. Thanks!

**Comment by souvik1997** [comment]
Would a790e31 be ok then? It passes the tests.

**Pull request closed without merging**